flight/Modules/Actuator/actuator.c

```diff
@@ -413,37 +414,56 @@ float ProcessMixer(const int index, const float curve1, const float curve2,
 	return(result);
 }
 
-
 /**
- *Interpolate a throttle curve. Throttle input should be in the range 0 to 1.
- *Output is in the range 0 to 1.
+ * Interpolate a mixer curve. Input is assumed to be in the range [0,1] unless otherwise specified.
+ * Output range is defined by the curve vector
+ * curve is defined in N intervals connecting N+1 points
```

Comment:
Maybe use Doxygen comments for the parameters? e.g.
```
 * @param[in] curve Curve is defined in N intervals connecting N+1 points
```
I know only some of the code is currently documented in this way but it would be nice to have more so we can generate nice code documentation. :)